src/MaterialDesignDemo.Shared/MaterialDesignDemo.Shared.shproj

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Though this approach works, I am thinking it might be simpler to just do a shared class library between the two projects.